core/jreleaser-model-impl/src/main/java/org/jreleaser/model/internal/validation/announce/BlueskyAnnouncerValidator.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2020-2023 The JReleaser authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jreleaser.model.internal.validation.announce;
+
+import org.jreleaser.bundle.RB;
+import org.jreleaser.model.internal.JReleaserContext;
+import org.jreleaser.model.internal.announce.BlueskyAnnouncer;
+import org.jreleaser.util.Errors;
+
+import java.nio.file.Files;
+
+import static org.jreleaser.model.api.announce.BlueskyAnnouncer.BLUESKY_ACCESS_TOKEN;
+import static org.jreleaser.model.internal.validation.common.Validator.checkProperty;
+import static org.jreleaser.model.internal.validation.common.Validator.resolveActivatable;
+import static org.jreleaser.model.internal.validation.common.Validator.validateTimeout;
+import static org.jreleaser.util.CollectionUtils.listOf;
+import static org.jreleaser.util.StringUtils.isBlank;
+import static org.jreleaser.util.StringUtils.isNotBlank;
+
+/**
+ * @author Simon Verhoeven
+ * @author Tom Cools
+ * @since 1.7.0
+ */
+public class BlueskyAnnouncerValidator {
+    private BlueskyAnnouncerValidator() {
+        // noop
+    }
+
+    public static void validateBluesky(JReleaserContext context, BlueskyAnnouncer announcer, Errors errors) {
+        context.getLogger().debug("announce.bluesky");
+
+        resolveActivatable(context, announcer, "announce.bluesky", "NEVER");
+        if (!announcer.resolveEnabledWithSnapshot(context.getModel().getProject())) {
+            context.getLogger().debug(RB.$("validation.disabled"));
+            return;
+        }
+
+        announcer.setHost(
+            checkProperty(context,
+                listOf(
+                    "announce.bluesky.host",
+                    "bluesky.host"),
+                "announce.bluesky.host",
+                announcer.getHost(),
+                errors));
+
+        announcer.setHandle(
+            checkProperty(context,
+                listOf(
+                    "announce.bluesky.handle",
+                    "bluesky.handle"),
+                "announce.bluesky.handle",
+                announcer.getHost(),
+                errors));
+
+        announcer.setPassword(
+            checkProperty(context,
+                listOf(
+                    "announce.bluesky.password",
```

Comment:
There may be an inconsistency with the chosen env var vs. the property name. This configuration resolves to [`JRELEASER_ANNOUNCE_BLUESKY_PASSWORD`, `JRELEASER_BLUESKY_ACCESS_TOKEN`].

it should be [`JRELEASER_ANNOUNCE_BLUESKY_PASSWORD`, `JRELEASER_BLUESKY_PASSWORD`].

This `BLUESKY_ACCESS_TOKEN` should  change to `BLUESKY_PASSWORD` (both key and value).